addons/binding/org.openhab.binding.somfytahoma/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,26 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: SomfyTahoma Binding
+Bundle-SymbolicName: org.openhab.binding.somfytahoma;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.2.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: .
+Import-Package: com.google.gson,
+ org.apache.commons.lang,
+ org.openhab.binding.somfytahoma,
```

Comment:
Could you please add a

```
org.eclipse.jdt.annotation;resolution:=optional,
```

as done in PR https://github.com/openhab/openhab2-addons/pull/2556